### ISSUE TITLE
Picture repetition rate is too high.

### ISSUE THREAD

**Issue by liuyulvv** [p2irc/rgb2nir#2]
From the data you provided, you can see that pictures with similar file names have a large area of ​​repetition. It is conceivable that the same is true for the validation set and the test set. This means that there are a large number of pictures of the same scene appearing in the training set and the test set at the same time. Have you considered this issue?

**Comment by bia006** [comment]
Thanks for the question. In our rgb2nir project, we translated rgb images to corresponding nir band. The trainA and trainB folders contain training samples in which each file in the trainA is translated to the same file in trainB folder. They should represent the same scene because our images were aligned pixel-wise. Our dataset contain aerial field images with self-similar pattern.

**Comment by liuyulvv** [comment]
Thank you for your answer, the images of RGB and NIR in training dataset should be the same, composed of pair. 

My confusion is whether the images of the test set will have a repeating scene with the training set?

For example, 1004 .png and 1005 .png, and if one of them appears in the training set and the other in the test set, the test set contains information about the training set because the area where the image repeats is high.

Have you thought about it?

**Comment by bia006** [comment]
Spatial overlap in raw aerial images is inevitable. But we tried to avoid it as much as possible. To create a test set, we selected images manually and mostly in a row which may have spatial overlap among test set images but not with training set. Also, the size of our original aerial images is about 960x1260 which would become a little smaller after image registration. We cropped random patches with the size of 256x256 for training and random cropped patches with the size of 512x512 for inferencing. By all of these steps, there is no overlap between training and test sets.
